src/db/mongo.py

```diff
@@ -6,8 +6,17 @@
 
 
 class MongoEngine(metaclass=Singleton):
+    """
+        Class that handle Mongodb connection
+    """
     def __init__(self):
+        """
+            Class constructor
+        """
         self.mongo_uri = environ.get('MONGO_URI')
 
     def get_connection(self):
+        """
+        Get mongo URL
+        """
```

Comment:
Advice:
Sometimes, write a simple comment there is not needed, your code should be understandable